include/drivers/gps.h

```diff
@@ -329,8 +329,7 @@ struct gps_driver_api {
 /**
  * @brief Function to start GPS operation.
  *
- * If gps is already running a call to this function will
- * restart the gps.
+ * If gps is already running a call to this function will fail.
```

Comment:
Return values should be documented.

Reply:
Added @return descriptions for `gps_start()` and `gps_stop()`. I didn't list possible return values as it hasn't been done for any of the functions and I think we should encourage moving to use the new GNSS API anyway.